spec/features/features_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'order'
+require 'menu'
+require 'calculate_bill'
+
+# As a customer
+# So that I can check if I want to order something
+# I would like to see a list of dishes with prices
+
+describe Order do
+  it 'starts a new order' do
+   order = Order.new
+  end
+
+  describe '#show_menu' do
+    it 'shows the list of available dishes' do
+     order = Order.new
+     order.show_menu
+    end
+  end
+
+# As a customer
+# So that I can order the meal I want
+# I would like to be able to select some number of several available dishes
+  
+  describe '#choose' do
+   it 'records the quantity as well' do
+    order = Order.new
+    order.choose("Margherita", 2)
+  end
+ end
```

Comment:
Lint/BlockAlignment: `end` at 30, 1 is not aligned with `describe '#choose' do` at 25, 2.